torch/fx/_symbolic_trace.py

```diff
@@ -205,6 +207,10 @@ def __init__(self, autowrap_modules: Tuple[ModuleType] = (math, ), enable_cpatch
 
                 Turning this on is likely to slow down tracing by 1.5-3x.
 
+            param_shapes_constant (bool): see https://github.com/pytorch/pytorch/issues/61733. When
```

Comment:
nit: Issues shouldn't be used as documentation. If there's information in the issue that is required for the user to understand the feature, that information should appear directly in the docstring